LogMonitor/src/LogMonitor/Utility.cpp

```diff
@@ -250,8 +250,9 @@ std::wstring Utility::ReplaceAll(_In_ std::wstring Str, _In_ const std::wstring&
 /// 
 /// helper function for a basic check if a string is a Number (JSON)
 /// as per the JSON spec - https://www.json.org/json-en.html
+/// only numbers not covered are those in scientific e-notation
 /// 
-bool Utility::isJsonNumber(_In_ PWSTR str)
+bool Utility::isJsonNumber(_In_ std::wstring& str)

```

Comment:
is this utility function used somewhere else in the project and does the reference need to be changed from taking PWSTR input to std::wstring?

Reply:
Only used in one place so far, in `EtwMonitor.cpp`; which actually necessitated the change.